homeassistant/components/playstation_network/media_player.py

```diff
@@ -0,0 +1,133 @@
+
+
+@dataclass
+class PSNdata:
+    """PSN dataclass."""
+
+    account = {"id": "", "handle": ""}
+    presence = {"availability": "", "lastAvailableDate": ""}
+    platform = {"status": "", "platform": ""}
+    title = {"name": "", "format": "", "imageURL": None, "playing": False}
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    config_entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Media Player Entity Setup."""
+    coordinator: PlaystationNetworkCoordinator = config_entry.runtime_data
+
+    if coordinator.data.platform.get("platform") is None:
+        username = coordinator.data.username
+        _LOGGER.warning(
+            "No console found associated with account: %s. -- Skipping creation of media player",
+            username,
+        )
+        return
+
+    async_add_entities([MediaPlayer(coordinator)])
+
+
+class MediaPlayer(PlaystationNetworkEntity, MediaPlayerEntity):
+    """Media player entity representing currently playing game."""
+
+    entity_description = MediaPlayerEntityDescription(
+        key="console",
+        translation_key="console",
+        device_class=MediaPlayerDeviceClass.TV,
+    )
+
+    def __init__(self, coordinator: PlaystationNetworkCoordinator) -> None:
+        """Initialize PSN MediaPlayer."""
+        super().__init__(coordinator)
+        self.psn: PlaystationNetworkData = self.coordinator.data
+        _attr_translation_key = "spotify"
+
+    @property
+    def media_image_remotely_accessible(self) -> bool:
+        """Is media image remotely accessible getter."""
+        return True
+
+    @property
+    def state(self) -> MediaPlayerState:
+        """Media Player state getter."""
+        match self.psn.platform.get("onlineStatus", ""):
+            case "online":
+                if (
+                    self.psn.available is True
+                    and self.psn.title_metadata.get("npTitleId") is not None
+                ):
+                    return MediaPlayerState.PLAYING
+                return MediaPlayerState.ON
+            case "offline":
+                return MediaPlayerState.OFF
+            case _:
+                return MediaPlayerState.OFF
+
+    @property
+    def unique_id(self) -> str:
+        """Unique ID Getter."""
+        return f"{self.psn.username.lower()}_{self.psn.platform.get('platform', "").lower()}_console"
+
+    @property
+    def name(self) -> str:
+        """Name getter."""
+        return f"{self.psn.platform.get('platform')} Console"
+
+    @property
+    def media_content_type(self) -> MediaType:
+        """Content type of current playing media."""
+        return MediaType.GAME
+
+    @property
+    def media_title(self) -> str | None:
+        """Media title getter."""
+        if self.psn.title_metadata.get("npTitleId"):
+            return self.psn.title_metadata.get("titleName")
+        if self.psn.platform.get("onlineStatus") == "online":
+            return "Browsing the menu"
```

Comment:
```suggestion
```

It will not be possible to translate this, not sure if this makes sense, but maybe return None if there is no title.